rancilio-pid/rancilio-pid.ino

```diff
@@ -932,12 +934,17 @@ void sendToBlynk() {
         blynksendcounter++;
       }
     }
-    if (blynksendcounter >= 5) {
+    if (blynksendcounter == 5) {
       if (String(Input, 2) != PreviousInputString) {
         Blynk.virtualWrite(V2, String(Input, 2)); //send value to server
         PreviousInputString = String(Input, 2);
       }
+      Blynk.virtualWrite(V3, setPoint);
       //Blynk.syncVirtual(V2);  //get value from server
+    }
+    if (blynksendcounter >= 6) {
+      power_off_minutes = power_off_timer/60 +  1;
```

Comment:
Ich finde es unnötig eine eigentlich statische Konfiguration remote einstellbar zu machen. Das verkompliziert nur. Daher würde ich das Weglassen, wenn du keine triftigen Gründe hierfür nennst.

Reply:
vermutlich willst du das auch in blynk angezeigt haben. ich bau es ein.

Reply:
ja, wollte nur in meiner blynk app sehen, wie lange die maschine noch läuft, bevor sie sich automatisch abschaltet (Silvia E)